templates/kisti_header.cfg

```diff
@@ -17,5 +17,5 @@ module add intel/18.0.3 impi/18.0.3 craype-network-opa craype-mic-knl
 ##export PYTHONPATH=$gmsim_root/Environments/nurion/workflow
 ##source $gmsim_root/Environments/nurion/virt_envs/python3_nurion/bin/activate
 export SLURM_JOB_ID="${PBS_JOBID/.pbs/}"
-export SLURM_NTASKS=$PBS_TASKNUM
+export SLURM_NTASKS=$(qstat $SLURM_JOB_ID -f | grep "used.ncpus" | cut -d"=" -f2 | xargs)
```

Comment:
why is it piped to xargs with no command?

Reply:
To trim the space off the front. Apparently that's the best way to do that.

Reply:
for this case using 'bc' should do the trick as well, it will also raise a error if the format has unexpected change and are no longer numbers from the 'cut command'.